assets/javascripts/discourse/templates/admin-plugins-gamification.hbs

```diff
@@ -52,9 +70,20 @@
       action=(action "saveEdit")
       disabled=saveEditDisabled
     }}
+    {{d-button
+      class="btn-secondary"
+      label="gamification.cancel"
+      title="gamification.cancel"
+      action=(action "saveEdit")
```

Comment:
Should the action here be something like `cancelEdit`?

Reply:
oops yeah huh, I def forgot to change this. thanks